### ISSUE TITLE
Hero Card not fully Support at Skype

### ISSUE THREAD

**Issue by mubat** [MicrosoftDocs/bot-docs#1600]
Please update information about support Hero Card at Skype - a lot of action type not support

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 60950444-cdcc-f0b9-aa21-c99894e26b50
* Version Independent ID: 0bd14221-e030-f9d8-41e6-1a52c9a76440
* Content: [Channels reference - Bot Service](https://docs.microsoft.com/en-us/azure/bot-service/bot-service-channels-reference?view=azure-bot-service-3.0#feedback)
* Content Source: [articles/bot-service-channels-reference.md](https://github.com/MicrosoftDocs/bot-docs/blob/live/articles/bot-service-channels-reference.md)
* Service: **bot-service**
* GitHub Login: @ivorb
* Microsoft Alias: **kamrani**

**Comment by mdrichardson** [comment]
@mubat I'll start doing some more extensive testing and update the chart. Do you mind sharing the code for the card you found not to be supported? I want to make sure my new testing incorporates your issue (and more).

**Comment by mdrichardson** [comment]
@mubat Disregard the request for your code. I made a PR that addresses this. Unfortunately, support for CardActions varies so much by channel there isn't really a good way to address this fully, so I settled on a more broad, "Card is supported fully with the exception that some channels only support a subset of CardActions and/or may limit the number of actions allowed on each card.  Varies by channel."

Thanks for calling this out!
